File: src/main/java/br/com/dusty/dkits/util/LocationUtils.java
package br.com.dusty.dkits.util;

import br.com.dusty.dkits.Main;
import org.bukkit.Location;

public class LocationUtils {
	
	public static Location spread(Location location, float range) {
		return location.clone().add((Main.RANDOM.nextFloat() - 0.5) * range, 0, (Main.RANDOM.nextFloat() - 0.5) * range);
	}
}

File: src/main/kotlin/br/com/dusty/dkits/util/LocationUtils.kt
package br.com.dusty.dkits.util

import br.com.dusty.dkits.Main
import org.bukkit.Location

fun Location.spread(range: Float): Location = this.clone().add((Main.RANDOM.nextFloat() - 0.5) * range, 0.0, (Main.RANDOM.nextFloat() - 0.5) * range)
